functions/gmail-notifier/index.js

```diff
@@ -76,6 +77,18 @@ exports.initWatch = async (req, res) => {
   return res.send(`Watch initialized on gmail inbox: ${email}`)
 }
 
+exports.postMessageToSlack = async (text) => {
```

Comment:
would be better to have test

Reply:
Would it be better to mock this function entirely ?
I do not see a good testing strategy if not mocking the slack API, which is basically all what this function call